react-main/src/redux/logout/reducer.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { actions } from './actions'
+
+const initialState = { token: '' };
+
+export function reducerLogout(state = initialState, action) {
+  switch (action.type) {
+    case actions.LOGOUT:
+      localStorage.clear();
```

Comment:
I'd do this side-effects in the action. The reducer should be limited to updating the state

Reply:
what about this?

Reply:
What about this?